Misalignment in Instructor table while adding Instructors by admin

**Environment**
master branch at 46a66b36cd744207ba277f5717aebf7edfacdd7c

**Steps to reproduce:**
- Sign in as admin
- Under `Create Instructor`, add an Instructor by providing name | email | Institute or by filling up the form 
- Check the table below that will appear after either the adding of Instructor is successful or fails 

**Screenshot below:**
**Before the commit**
![screenshot 330](https://user-images.githubusercontent.com/10681755/35652652-6f7b4ada-06dc-11e8-998c-c7f0b00ba023.png)

**After**
![screenshot 331](https://user-images.githubusercontent.com/10681755/35652660-7e973ce0-06dc-11e8-8c75-24e5b31ebae4.png)

**Expected behaviour:** Misalignment should not occur.

**Actual behaviour:** Misalignment is occuring and due to that the `AdminHomePageUiTest` is failing repeatedly on my system, ( Though I have not seen this test failing in any Travis CI tests)
So wanted to confirm if anyone else is getting this bug or is it me who is missing something.  

Also As I am not sure if it is a bug or error in my side, I will provide the error I am getting with the screenshot also. 

```xml
          <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<Instructor Details must have 3 columns> but was:<Fail>
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
	at teammates.test.cases.BaseTestCase.assertEquals(BaseTestCase.java:142)
	at teammates.test.cases.browsertests.AdminHomePageUiTest.testCreateInstructorAction(AdminHomePageUiTest.java:84)
	at teammates.test.cases.browsertests.AdminHomePageUiTest.testAll(AdminHomePageUiTest.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
```

**Screenshot:** This is where the test is failing (Apparently due to the misalignment)
![screenshot 326](https://user-images.githubusercontent.com/10681755/35652901-9b177122-06dd-11e8-9628-36b2adb518ff.png)

--- a/src/main/webapp/dev/js/common/instructorFeedbackResults.js
+++ b/src/main/webapp/dev/js/common/instructorFeedbackResults.js
@@ -242,7 +242,7 @@ function getAppendedResponseRateData(data) {
 }
 
 function toggleNoResponsePanel(e) {
-    const $targetElement = $(e.currentTarget);
+    const $targetElement = $(e.target);
     if ($targetElement.is('a') || $targetElement.is('input')) {
         return;
     }

--- a/src/main/webapp/dev/js/main/instructorHome.js
+++ b/src/main/webapp/dev/js/main/instructorHome.js
@@ -51,7 +51,7 @@ function bindCoursePanels() {
  * Changes the state of the course panel (collapsed/expanded).
  */
 function toggleCourseVisibility(e) {
-    const $targetElement = $(e.currentTarget);
+    const $targetElement = $(e.target);
     if ($targetElement.is('a') || $targetElement.is('input') || $targetElement.hasClass('dropdown-toggle')) {
         return;
     }

